resources/mac/helpers/org.tidepool.disk-permissions/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+BASE_PATH="$(cd "$(dirname "${0}")"; pwd)"
+HELPER_SCRIPT="org.tidepool.disk-permissions.sh"
+HELPER_SCRIPT_PATH="/Library/PrivilegedHelperTools"
+PLIST_NAME="org.tidepool.disk-permissions.plist"
+PLIST_PATH="/Library/LaunchDaemons"
+
+if [ "${1}" = "--uninstall" ]; then
```

Comment:
The more standard way of parsing args is using `case`. Not a blocker.

Reply:
I agree that `case` would be better, but let's make that change if/when we add more than two args.